web/package.json

```diff
@@ -30,6 +30,7 @@
     "axios": "^0.27.2",
     "date-fns": "^2.28.0",
     "framer-motion": "^6.3.4",
+    "playwright": "^1.28.0",
```

Comment:
be sure not to include playwright as a standard dependency because it's already included as a dev dependency